sorted_db/db.go

```diff
@@ -193,6 +261,64 @@ func (db *DB) Search(needle []byte) []byte {
 	return nil
 }
 
+// Retrieves all records that have keys starting with needle.
```

Comment:
Linter says: `Comment on exported method DB.ForwardMatch should be of the form "ForwardMatch ..."`